webhooks.md

```diff
@@ -503,6 +503,59 @@ curl -X POST \
   }'
 ```
 
+`transaction.reimbursement.status` event is triggered when a reimbursement changes state on a transaction.
+
+The payload for these events includes the `reimbursement` object with the up-to-date status, and you can read more about it in the [Reimbursement API](/reimbursement/#webhook) documentation.
+```json
+"reimbursement": {
+    "amount": 2.55,
+    "created": "2021-09-30T11:11:11.000Z",
+    "creditsTransactionId": "1250ab5a-0661-4a06-a40c-8514093a9241",
+    "description": "Earned Stars",
+    "status": "issued",
+    "token": "6c01f956-1f0f-413f-a5db-d1fc8a59ef92",
+  }
+```
+
+### Credits
+Credits is a supporting product for reimbursement, you can learn more about them [here](/reimbursement/#credits). A `credits.balance.low` event is sent when a currency's credit balance drops below 25% of the amount at the time of the previous credits purchase. The payload for these events are the following:
```

Comment:
"Credits is a supporting product for reimbursement" - this sounds like you are trying to sell it, not trying to explain it :-) 
I would try to explain very shortly what they are: "You can use Fidel API Credits to reimburse cardholders. Learn more about Fidel API Credits on the [Reimbursement page]"

Also, please clarify with your PM how they should be called, "Fidel API Credit", or "Fidel Credit", do we have to use plural, or even better: can we just say credit? This text doesn't use this term consequently.

Plus: "The payload for these events is the following:"

Reply:
Also updated this section. In the Reimbursement page (https://fidelapi.com/docs/reimbursement/#credits) it's called "Fidel Credits" so I kept that name here as well, to be consistent